Scoop/Controller/LocationViewController.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  LocationViewController.swift
+//  Scoop
+//
+//  Created by Elvis Marcelo on 4/27/22.
+//
+
+import UIKit
+
+protocol LocationViewControllerDelegate: AnyObject {
+    func didSelectLocation(location: String)
+}
+
+class LocationViewController: UIViewController {
+    
+    private var tableView = UITableView()
+    private var locations:[String] = []
+    private var filteredLocations:[String] = []
+    private let reuseIdentifier = "locationCellReuse"
+    private let cellHeight: CGFloat = 50
+    
+    weak var delegate: LocationViewControllerDelegate?
+    
+    private var searchController: SearchInitialViewController!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupTableView()
+    }
+    
+    init(searchController: SearchInitialViewController) {
+        super.init(nibName: nil, bundle: nil)
+        self.searchController = searchController
```

Comment:
I think if you initialize `self.searchController` above the `super.init`, it doesn't have to be an optional?